### ISSUE TITLE
It should be possible to disable injection of script AssemblyAttribute

### ISSUE THREAD

**Issue by oleg-shilo** [oleg-shilo/cs-script#14]
The injected attribute contains the path of the original script file being executed. Useful for debugging and troubleshooting.
 
The are a few minor problems associated with injection of script assembly attributes. 
- The settings that enables/disables it is not publicly exposed.
- The assembly attribute is injected even for LoadCode use cases. What is not useful at all the stored path is a temp (artificial) script file.

**Comment by oleg-shilo** [comment]
Fixed with commit 7b5bfd9bf316b2dd2b24a9ccc55c524074b4811a
Will be available with the next release.
___
Now you can disable the injection as below:
```C#
CSScript.GlobalSettings.InjectScriptAssemblyAttribute = false;
```
And injection is automatically disabled for the "fake" scripts generated by LodCode calls.

**Comment by oleg-shilo** [comment]
Fixed (Release v3.16).
Thank you. 
https://github.com/oleg-shilo/cs-script/releases/tag/v3.16.0.0
https://chocolatey.org/packages/cs-script/3.16.0.0
